GitUI/Editor/FileViewer.cs

```diff
@@ -71,11 +71,11 @@ public FileViewer()
 
             IsReadOnly = true;
 
-            this.encodingToolStripComboBox.Items.AddRange(new Object[]
-                                                    {
-                                                        "Default (" + Encoding.Default.HeaderName + ")","DOS852", "ASCII",
-                                                        "Unicode", "UTF7", "UTF8", "UTF32"
-                                                    });
+            var encodingArray = (from e in AppSettings.AvailableEncodings.Values
+                                select e.EncodingName).ToArray();
```

Comment:
this should be sorted in some meaningful manner.
maybe even group by dos, windows, mac, etc

```
--- DOS ---
encoding1 (DOS)
encoding2 (DOS)
-- Windows ---
encoding1 (Win)
encoding2 (Win)
```